### PR TITLE
Add new output plugin for Seq

### PR DESCRIPTION
**Pull request by nskerl**
This output plugin sends Telegraf metrics as structured log events to a [Seq](https://getseq.net/) instance.

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

File: plugins/outputs/seq/seq.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
+package seq
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/influxdata/telegraf"
```
Comment (glinton) [inline plugins/outputs/seq/seq.go]: nitpick: separate stdlib and other imports

File: plugins/outputs/seq/seq.go [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+		if err != nil {
+			return []byte{}, err
+		}
+		se["Host"] = host
+	}
+
+	se["Tags"] = m.Tags()
+	se["Fields"] = m.Fields()
+
+	serialized, err := json.Marshal(se)
```
Comment (glinton) [inline plugins/outputs/seq/seq.go]: Just `return json.Marshal(se)`

File: plugins/outputs/seq/seq.go [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+func (a *Seq) Write(metrics []telegraf.Metric) error {
+	if len(metrics) == 0 {
+		return nil
+	}
+
+	var buffer bytes.Buffer
+
+	for _, metric := range metrics {
+
```
Comment (glinton) [inline plugins/outputs/seq/seq.go]: nitpick: no empty line here

File: plugins/outputs/seq/seq.go [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+		if err != nil {
+			return err
+		}
+
+		buffer.Write(line)
+		buffer.WriteString("\n")
+	}
+
+	req, err := http.NewRequest("POST", a.authenticatedUrl(), &buffer)
+
```
Comment (glinton) [inline plugins/outputs/seq/seq.go]: nitpick: no empty line here

File: plugins/outputs/seq/seq_test.go [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConnectAndWrite(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping integration test in short mode")
+	}
+
+	url := testutil.GetLocalHost() + ":5341"
```
Comment (glinton) [inline plugins/outputs/seq/seq_test.go]: Need to add scheme (`http`) and don't forget to start the server somewhere in the tests too.

File: plugins/outputs/seq/seq.go (hunk 1 above)
Comment (nskerl) [reply plugins/outputs/seq/seq.go]: Fixed.

File: plugins/outputs/seq/seq.go (hunk 2 above)
Comment (nskerl) [reply plugins/outputs/seq/seq.go]: Fixed.

File: plugins/outputs/seq/seq.go (hunk 3 above)
Comment (nskerl) [reply plugins/outputs/seq/seq.go]: Fixed.

File: plugins/outputs/seq/seq.go (hunk 4 above)
Comment (nskerl) [reply plugins/outputs/seq/seq.go]: Fixed.

File: plugins/outputs/seq/seq_test.go (hunk 5 above)
Comment (nskerl) [reply plugins/outputs/seq/seq_test.go]: Fixed. I refactored the test to use httptest.

**Comment by nskerl** [comment]
Thanks for the feedback @glinton .  Would you like me to create an issue as well?

**Comment by nskerl** [comment]
@glinton thanks for the feedback

**Comment by nskerl** [comment]
@glinton What are the next steps to get this pulled? Is it a candidate for 1.10?

**Comment by nskerl** [comment]
Thanks. @russorat please let me know when you have a chance to review.

**Comment by nskerl** [comment]
@russorat  Thanks again for reviewing our plugin. Please let me know if you have any advice on getting this included in 1.10 or future releases.

**Comment by russorat** [comment]
@nskerl thank you for the plugin. For new plugins to specific software vendors, we usually leave PRs open for some time in order to gather feedback from the community. We'd like to see comments and feedback from people using this output before we consider merging it to master. We will keep this open so people can find it and use it.

**Comment by nskerl** [comment]
@russorat Thanks! I'll try to get some feedback from users. I know of two others using builds off my branch.

**Comment by sjwang90** [comment]
@nskeri Checking in as it's been awhile. This output plugin actually looks like a great candidate for an [external plugin](https://github.com/influxdata/telegraf/blob/master/docs/EXTERNAL_PLUGINS.md) that can be used with `execd` to run seamlessly with Telegraf? 

Closing this for now. Telegraf users would be able to use the plugin sooner by having it as an external plugin since it wouldn't have to go through the typical review process as well.

**Pull request closed without merging**